packages/office-ui-fabric-react/src/utilities/grid/GridCell.Props.ts

```diff
@@ -0,0 +1,62 @@
+export interface IGridCellProps {
+
+  /**
+   * The option that will be made available to the user
+   */
+  item: any;
```

Comment:
If we made it item: <T>, the caller could say its <IColorCellProps> and then the typescript compiler would be able to confirm that you are dealing with IColorCellProps for the onRender function in the consumers.